docs/usage.md

```diff
@@ -121,3 +121,7 @@ Action Scheduler will register its version on `'plugins_loaded'` with priority `
 It is recommended to load it _when the file including it is included_. However, if you need to load it on a hook, then the hook must occur before `'plugins_loaded'`, or you can use `'plugins_loaded'` with negative priority, like `-10`.
 
 Action Scheduler will later initialize itself on `'init'` with priority `1`.  Action Scheduler APIs should not be used until after `'init'` with priority `1`.
+
+### Usage in Themes
+
+When using Action Scheduler in themes, it's important to note that the latest version of Action Scheduler registered by a plugin will be used rather than the version included in the theme. This is because of the version dependency handling code using `'plugins_loaded'` since version 1.0.
```

Comment:
Wording change suggestion

> it's important to note that if Action Scheduler has been registered by a plugin then the latest version registered by a plugin will be used rather ...

Reply:
Changed to that and rebased in the change. 👍